### ISSUE TITLE
linux p7zip 17.02 and end-user question

### ISSUE THREAD

**Issue by Choons** [techno-express/node-7z-archive#4]
Hi, I have been looking for something like this for a while now. Couple of questions: where does one get the linux p7zip v. 17.02, and do end-users have to have 7-zip installed for the SFX to extract?

**Comment by techno-express** [comment]
The readme states 7zip is download at `npm install` time and shows the location it's coming from, will be within node_modules folder. And SFX means the actual extracting code is embedded, no 7-zp needed.

**Comment by Choons** [comment]
Yeah I see that, but p7zip last version on sourceforge is 16.02, not 17.02. I know there's a fork on here releasing newer numbered versions, but I don't know how official or safe that is. 

Glad to hear you define SFX the way it's meant to be-- a lot of other folks promise SFX and then it turns out the end user has to jump through a bunch of hoops & install their plugin etc. before it will extract and execute.

Thanks for making node-7z-archive. If it does what I think, it should be great for web apps keeping everything client side and allowing people to share their creations.

**Comment by techno-express** [comment]
You might want to visit the 7zip/p7zip forums about the official status. Also visit 7-zip.org website see where original author direct users for anything other than the Windows, the only official version. There is an issue on https://github.com/jinfeihan57/p7zip/issues about this also.

**Comment by Choons** [comment]
Will do, thanks for the leads as I was wondering what's up. I used 7z years ago on Windows before I switched to linux and it was undoubtedly the best at creating tiny archives fast. I was hoping it was still a viable project. Looks like you are helping it to be so!
